tests/observability/metrics/test_migration_metrics.py

```diff
@@ -273,9 +286,16 @@ def test_kubevirt_vmi_migration_metrics(
         vm_migration_metrics_vmim_scope_class,
         query,
     ):
+        time_passed_from_starting_migration = (
```

Comment:
since you are dividing by 60 - how about more declerative name?
```suggestion
        minutes_passed_since_migration_start = (
```\